src/main/native/cpp/CvSinkImpl.cpp

```diff
@@ -1,22 +1,21 @@
 /*----------------------------------------------------------------------------*/
-/* Copyright (c) FIRST 2016. All Rights Reserved.                             */
+/* Copyright (c) 2016-2017 FIRST. All Rights Reserved.                        */
 /* Open Source Software - may be modified and shared by FRC teams. The code   */
 /* must be accompanied by the FIRST BSD license file in the root directory of */
 /* the project.                                                               */
 /*----------------------------------------------------------------------------*/
 
 #include "CvSinkImpl.h"
 
-#include "llvm/SmallString.h"
-#include "opencv2/core/core.hpp"
-#include "opencv2/imgproc/imgproc.hpp"
-#include "opencv2/highgui/highgui.hpp"
-
-#include "cscore_cpp.h"
-#include "c_util.h"
 #include "Handle.h"
 #include "Log.h"
 #include "Notifier.h"
+#include "c_util.h"
+#include "cscore_cpp.h"
+#include "llvm/SmallString.h"
+#include "opencv2/core/core.hpp"
+#include "opencv2/highgui/highgui.hpp"
+#include "opencv2/imgproc/imgproc.hpp"
```

Comment:
The opencv includes are part of a different project entirely, so they should use angle brackets.